### ISSUE TITLE
Why is the first project's title part of the global atom tab pane?

### ISSUE THREAD

**Issue by rondonjon** [Cirex/atom-metro-ui#13]
I find it confusing that the title bar of the first project in the tree view is part of the atom application tab pane. (Is that because of the Metro UI theme or because atom wants it this way?)

If at least the title bars of the other projects had the same color. In my case, the topmost project's title bar has a strong blue background color, which makes it look "active" compared to the following projects with their grey title bar backgrounds.

![titlebar](https://cloud.githubusercontent.com/assets/8258823/9735851/ea00dbe6-563d-11e5-8b96-218d6fd06601.jpg)

**Comment by Cirex** [comment]
I agree with you on it being a little confusing. In the original mock up the top portion of the blue bar in the tree-view was occupied by a work space/project switcher that i've been slowly working on as time permits, and every project header (including the top most) was meant to be colored the grey.

**Comment by rondonjon** [comment]
How about an ATOM logo in the top section until a better use is found?

**Comment by Cirex** [comment]
I don't know about that one. I've always been a minimalist and having a logo that serves no purpose just seems like a waste of space to me. I'll have to play around and see what I can come up with
